lib/pro/set-status.js

```diff
@@ -13,13 +14,28 @@ function setStatusPro (newStatus, context) {
   const { repository, pull_request: pullRequest } = context.payload
   const { name } = getAppConfig()
 
+  let output = null
+
+  let emojiFound = false
+
+  output = emojiText.convert(newStatus.match, {
+    callback: function (emoji, data) {
+      emojiFound = true
+      return `a ${data.description.toLowerCase()} emoji!`
+    }
+  })
+
+  if (!emojiFound) {
+    output = `"${newStatus.match}"`
+  }
+
```

Comment:
With gemoji, we could do something like this:

```js
if (gemoji.unicode[newStatus.match]) {
  newStatus.match = gemoji.unicode[newStatus.match].name + ' emoji'
}
```

I think overriding `newStatus.match` might be simpler than using a new variable called `ouptup`, what do you think?

Now one thing that might still cause trouble is if people set terms that are more than just one emoji or something like "WIP 🚧". But it’s an edge case that didn’t occur yet as far as I can tell.

Reply:
an alternative would be to use `gemoji` together with `emoji-regex`

```js
const gemoji = require('gemoji')
const emojiRegex = require("emoji-regex")

newStatus.match = newStatus.match.replace(emojiRegex(), (match) => gemoji.unicode[match].name + ' emoji')
```

Reply:
Thanks for the feedback. The reason I created `output` was so the details section would still contain the emoji.
![image](https://user-images.githubusercontent.com/15106748/49700388-c05b7a00-fbab-11e8-9b8e-71792f03d2f8.png)
Would you rather it show the emoji or the name?

Reply:
> an alternative would be to use `gemoji` together with `emoji-regex`
> 
> ```js
> const gemoji = require('gemoji')
> const emojiRegex = require("emoji-regex")
> 
> newStatus.match = newStatus.match.replace(emojiRegex(), (match) => gemoji.unicode[match].name + ' emoji')
> ```

@HaydenMeloche what do you think about this ☝️